src/WatchDogs.Test/DxTradeAuthenticatorTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Contracts;
+using Infrastructure.DxTrade;
+using Microsoft.Extensions.Options;
+using Moq;
+using RichardSzalay.MockHttp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WatchDogs.Test;
+public class DxTradeAuthenticatorTests
+{
+    private readonly Mock<IHttpClientFactory> _httpClientFactoryMock = new Mock<IHttpClientFactory>();
+    private readonly Mock<ISessionTokenStorage> _sessionTokenStorageMock = new Mock<ISessionTokenStorage>();
```

Comment:
As we discussed, you can remove the mock from here and also from `DxTradeAuthenticator`